### PR TITLE
Aleksei/analytics fix

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

Fixed data sent to analytics. Also, set numeric values to an integer by multiplying to 1000 and rounding.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -710,13 +709,15 @@ export default {
...
+          network: this.network.id,
+          address: this.session.address
         },
         "Action",
+        "Modal",
         this.featureFlag,
-        "",
-        this.amount
+        this.featureFlag == "claim_rewards"
+          ? this.rewards[0].amount
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue]: this will not work with multi denom rewards 🤷‍♂

Comment (iambeone) [reply src/ActionModal/components/ActionModal.vue]: We will fix it when we will have multi denoms)

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: we already have multi denom rewards

Comment (jbibla) [reply src/ActionModal/components/ActionModal.vue]: > we already have multi denom rewards

only on e-Money which is a testnet

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: but we want to bring to mainnet asap to finish the milestone?!

**Pull request merged**